taskcluster/android_taskgraph/transforms/build_aab.py

```diff
@@ -0,0 +1,129 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Apply some defaults and minor modifications to the jobs defined in the
+build-bundle (build AAB) kind.
+"""
+
+from android_taskgraph.build_config import get_variant
+
+from taskgraph.transforms.base import TransformSequence
+from taskgraph.util.schema import resolve_keyed_by
+
+
+transforms = TransformSequence()
+
+
+@transforms.add
+def add_variant_config(config, tasks):
```

Comment:
Nit: If I'm not mistaken, we can probably factorize most of the transforms in a dedicated one because we duplicated the ones in

https://github.com/mozilla-mobile/firefox-android/blob/1336c4be3e1b5aeda8f5c21339aa3bc205efee36/taskcluster/android_taskgraph/transforms/build_apk.py#L18

Reply:
Yes, thanks. In fact, with just slight modifications we can re-use the build_apk transform for aab - much simpler.